src/main/java/frc/robot/commands/ShooterOnCommand.java

```diff
@@ -32,38 +29,20 @@ public void initialize() {
   @Override
   public void execute() {
 
-    double currentSpeed = m_shooterSubsystem.getShooterSpeed();
-    if (currentSpeed < 4500){ //subject to tuning
-      m_shooterSubsystem.setShooter(1);
-    }
-    
-    else if (currentSpeed > 4500) { //subject to tuning
-      m_shooterSubsystem.setShooter(0.85);
+    double currentSpeed = m_shooterSubsystem.getFlywheelRPM();
+    if (currentSpeed < Constants.ShooterConstants.FLYWHEEL_RPM_THRESHOLD) { // TODO subject to tuning
+      m_shooterSubsystem.setFlywheel(1);
     }
 
-    // m_pidController.setSetpoint(0.96);
-    // double pidOutput = m_pidController.calculate(m_shooterSubsystem.getShooterSpeed());
-    // if (pidOutput > 1) {
-    //   m_shooterSubsystem.setShooter(1);
-    // }
-    // m_shooterSubsystem.setShooter(pidOutput);
-    
-    
-
-    
-
-    // if (timer.hasElapsed(4)) {
-    //   m_shooterSubsystem.setShooter(0.8);
-
-    // } else {
-    //   m_shooterSubsystem.setShooter(0.1);
-    // }
-        
+    else if (currentSpeed > Constants.ShooterConstants.FLYWHEEL_RPM_THRESHOLD) { 
+      m_shooterSubsystem.setFlywheel(0.85);
```

Comment:
What should I call the constant that represents the number 0.85?  FLYWHEEL_LOWER_SPEED/POWER?